### PR TITLE
Minimum cost to generate N cells in C++

### PR DESCRIPTION
**Pull request by gayatri517**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Code added in cp(competitive programming) folder.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4983

### CODE REVIEW

**Comment by gayatri517** [comment]
@HarshCasper Kindly review my PR.
Thank you.

**Comment by gayatri517** [comment]
@ankitaggarwal23 please take a look at my PR

File: C-Plus-Plus/README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@
 - [Is Valid Sudoku](cp/Is_Valid_Sudoku.cpp)
 - [Find all unique prime factor of any number n](cp/Unique_prime_factors.cpp)
 - [Plus One Problem](cp/Plus_One.cpp)
+- [Minimum cost to generate n cells in the container starting from one cell](cp/min_cost_to_generate_n_cells.cpp)
```
Comment (HarshCasper) [inline C-Plus-Plus/README.md]: Keep it `Minimum Cost to generate N Cells`

File: C-Plus-Plus/cp/min_cost_to_generate_n_cells.cpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+/*Example: 
+1)Input: 5 2 1 3
+Output: 4
+2)Input: 7 4 2 3
+Output: 10
+Time Complexity: O(n)
+Space Complexity: O(n) */
+
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/min_cost_to_generate_n_cells.cpp]: Remove extra line

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by gayatri517** [comment]
@HarshCasper I have made the changes, Kindly review them

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**